QR Code Metrics

arjunsudheer opened the pull request:
Add custom Prometheus metrics for qr codes.

arjunsudheer committed:
Add custom qr_code_cache_size Prometheus metric

Use Gauge to track how many qr codes are being stored

arjunsudheer committed:
Add custom qr_code_cache_size_in_bytes Prometheus metric

Use Gauge to track the total size in bytes of all stored qr codes

evanugarte commented on modules/qr_code.py:
```diff
@@ -76,7 +88,11 @@ def delete(self, alias: str):
+            MetricsHandler.qr_code_cache_size_in_bytes.dec(os.path.getsize(path))
             os.remove(path)
+            # Decrease the qr_code_cache_size custom prometheus metric by 1 after a QR Code is removed
+            MetricsHandler.qr_code_cache_size.dec(1)
```
just to be certain
```suggestion
            MetricsHandler.qr_code_cache_size.set(len(self.mapping))
```

arjunsudheer replied:
Done

arjunsudheer committed:
Update qr_code.py after code review

Use set instead of dec for qr_code_cache_size in delete function to increase metric accuracy
